flowzone.yml

```diff
@@ -3184,7 +3184,8 @@ jobs:
 
   cloudformation_test:
     name: Test CloudFormation
-    runs-on: ${{ fromJSON(inputs.cloudformation_runs_on || inputs.runs_on) }}
+    # FIXME: https://github.com/actions/setup-python/issues/705
+    runs-on: ["ubuntu-22.04", "X64"]  # ${{ fromJSON(inputs.cloudformation_runs_on || inputs.runs_on) }}
```

Comment:
This tag combination isn't valid, it will look for a runner with both `ubuntu-22.04` (GitHub hosted) and `X64` which AFAIK only exists on self-hosted.

IMO we shouldn't be hardcoding runners, but instead should skip steps when incompatible runners are detected.